src/styles/Header.scss

```diff
@@ -87,4 +88,73 @@ header {
       }
     }
   }
+  .settings-wrapper{
```

Comment:
add empty line above

Reply:
![image](https://user-images.githubusercontent.com/41084902/115738125-6da64580-a395-11eb-8ca3-a2fe58aa5646.png)

Reply:
fixed

Reply:
good, this file has 2 spaces as expected